src/dataflow/src/sink/tail.rs

```diff
@@ -7,7 +7,11 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use timely::dataflow::channels::pact::Pipeline;
+use std::collections::BinaryHeap;
+
+use differential_dataflow::AsCollection;
+use differential_dataflow::{operators::consolidate::Consolidate, Hashable};
```

Comment:
super minor nit: I thought we were purposefully avoiding this import format?

Reply:
Were we? I was never purposefully doing anything; I just write code and tell my editor to automatically add the correct things to imports.

Reply:
In general I'm not a fan of making formatting changes that aren't enforced by an automated tool like `cargo fmt`; just seems like a waste of time. But, I would begrudgingly do so if we have a consensus that it's required; I just haven't seen one.

Reply:
that was my impression, could be off and I don't have strong feelings on the import format

Reply:
I don't believe that it's a hard requirement, cargo fmt doesn't currently support requiring a specific import style and our policy is, I believe, that if style isn't automatically enforceable then the most it can be is a preference.

So, yeah, I believe that most people (I know that I have this preference) don't do nested imports, especially if the imports break across lines. For me it is because (due to the state of rust tooling at the moment) it is nice to be able to do `rg differential.*Hashable` to find all uses, and if this import became longer and went into vertical mode we'd get:

```rust
 use differential_dataflow::{
    operators::consolidate::Consolidate, 
    Hashable,
};
```

that would stop being possible.

Again, afaik this is just a preference not a hard rule in code reviews, but the preferred style is one module per line.